src/common/views/opds.ts

```diff
@@ -144,4 +143,11 @@ export interface IOpdsNavigationLink {
     self: IOpdsLinkView[];
 }
 
-export type THttpGetOpdsResultView = THttpGetResultAfterCallback<IOpdsResultView>;
+// https://datatracker.ietf.org/doc/html/rfc7807#section-3.1
+export interface IOpdsProblemDetailsResultView {
```

Comment:
Why not `interface IOpdsProblemDetailsResultView extends IProblemDetailsResultView` ... or even `type IOpdsProblemDetailsResultView = IProblemDetailsResultView` given that there is currently no difference?

Reply:
forgot to remove it